src/test/shell/bazel/bazel_sandboxing_cpp_test.sh

```diff
@@ -200,6 +241,6 @@ EOF
 
 # The test shouldn't fail if the environment doesn't support running it.
 check_supported_platform || exit 0
-check_sandbox_allowed || exit 0
+check_sandbox_allowed || "${IS_WINDOWS}" || exit 0
```

Comment:
Sure you need this? The test is tagged `no_windows`.

Reply:
The test has `no_windows` tag because CI does not have the windows-sandbox binary, so can't run it by default. `check_sandbox_allowed` tries to check if `linux-sandbox` exists and valid which will fail on Windows. I could change `check_sandbox_allowed` to support Windows too, but that will make other sandbox `sh_test` to run on Windows and won't pass (if user invoke them manually).

Reply:
Never mind, I got confused over the logic here.